src/main/scala/uk/gov/nationalarchives/dp/client/Entities.scala

```diff
@@ -1,27 +1,37 @@
 package uk.gov.nationalarchives.dp.client
 
 import cats.MonadError
+import cats.implicits.catsSyntaxOptionId
 
 import java.util.UUID
 
 object Entities {
-  case class Entity(entityType: String, ref: UUID, title: Option[String], deleted: Boolean, path: String)
+  case class Entity(
+      entityType: Option[String],
```

Comment:
1. Should we have a default of `None` for this?
2. (Not really related to your new changes but) Should we add defaults for the other properties in this case class?

Reply:
I'm not sure it's worth it. To work out whether we have to put something in the case class, we have to check the XML response anyway which will come back with `Option("SO")` or None or whatever so we may as well pass it in.

What I think will happen is that this case class will mostly not be created by any calling code. It will be returned from the client, maybe passed around as json and then sent back into another client method so we don't really need defaults to make it easier on the developer.